### ISSUE TITLE
Group: Invite flow - pop up join group button has incorrect text

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#783]
In the flow of accessing the group via invite, it went something like: 

- Clicks join in email
- Registers
- Logs in
- Then gets the pop up telling him to "log in to join topic": 
![image](https://github.com/citizenos/citizenos-fe/assets/61320526/56553b2a-7ac6-41e8-9b1e-3e50f791e6e1)

Takes him to the dashboard (not the group, as per other issue already made) without asking him to log in again. 

So basically the button says the wrong thing:
- Shouldn't say log in (he was already)
- Should say join group (not topic)

**Comment by ssin1901** [comment]
@BeccaMelhuish good catch! No login button appeared, so moving it to Done (see screen recording)

https://github.com/user-attachments/assets/54603485-ba76-4fb1-89f1-17fc6e7cd2ec

**Comment by BeccaMelhuish** [comment]
@ssin1901 Ah though did you also test the flow for a group invite? As that's where the issue arose. Looks from the video that it's a topic invite? :)

**Comment by ssin1901** [comment]
@BeccaMelhuish my apologies, will test with Group as well.

**Comment by ssin1901** [comment]
@BeccaMelhuish can you check the recordings in the folder 
https://drive.google.com/drive/folders/1aMjUP1nqZSOaScWr5KIOsB_M5i2LaCd2?usp=sharing

Following scenario was recorded
1. User created an account from a group invite
2. Was prompted to login and then taken directly to the group (Login to join group text displayed)

Looks okay to me.

**Comment by BeccaMelhuish** [comment]
Looks OK yes thank you! :)
